rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/Fixture/alias_used_with_last_keyword.php.inc

```diff
@@ -1,37 +0,0 @@
-<?php
```

Comment:
I will look up on these removed tests and make it work again, ensuring no regression in the future, it seems it missing target stubs in exact source file

Reply:
https://github.com/rectorphp/rector-src/pull/6955